### ISSUE TITLE
[RTL] Highfi Topics In Progress / OngoingTopicList

### ISSUE THREAD

**Issue by rt4914** [oppia/oppia-android#2145]
`TopicsInProgress` or `OngoingTopicList` needs RTL support as shown in below screen.
![UP (Topics in Progress)](https://user-images.githubusercontent.com/9396084/99870699-3f141b00-2bfb-11eb-9a13-38a1317ab227.png)

Make sure that this works correctly in mobile (landscape + portrait) and tablet (landscape + portrait).

## References
Mobile Portrait: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/grid/
Mobile Landscape: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/grid/
Tablet: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/grid/
Wiki: https://github.com/oppia/oppia-android/wiki/Working-on-UI

**Comment by ForgottenProgramme** [comment]
Hey, @rt4914! 
I want to work on this. But, would you please give me some pointers on how to go about it?

**Comment by rt4914** [comment]
> Hey, @rt4914!
> I want to work on this. But, would you please give me some pointers on how to go about it?

I suggest having a look at following documents as they are really helpful:
https://android.jlelse.eu/rtl-support-on-android-here-is-all-you-need-know-e13f2df512e2
https://developer.android.com/training/basics/supporting-devices/languages?hl=ru#MirroringUpdateResources

**Comment by ForgottenProgramme** [comment]
Hey, @rt4914 . 
Thankyou very much for sharing those links. Rishabh Harit's article on RTL support was very informative and helpful. 

I went through several articles and stackoverflow queries and found that to add RTL support for any layout, I should add, android:textAlignment="viewStart"  in the textviews and also change the gravity from "top" to "start". 
I didn't quite believe it could be this simple, but I tried nonetheless. 

When I made those changes in the ongoing_topic_item.xml file in the layout folder alone, it didn't work for both mobile and tablet.
I also had to make those changes in layout-land, layout-sw600dp-land and layout-sw600dp-port folder.
